The URL for downloading .NET Framework 4.7.2 is not working

rramoscabral opened the pull request:
The URL https://go.microsoft.com/fwlink/?LinkID=863255 for downloading .NET Framework 4.7.2 is not working. This URL is being used in:
   - Line 11
   - Line 28
   - Line 30

I changed all .NET download URLs equal to the URL of the line 26.

Other URL:
 - https://www.microsoft.com/net/download?utm_source=ms-docs&utm_medium=referral (used on line 26)
 - https://dotnet.microsoft.com/download/dotnet-framework-runtime

## Summary

Describe your changes here.

Fixes #Issue_Number (if available)

rramoscabral committed:
The URL for downloading .NET Framework 4.7.2 is not working

The URL https://go.microsoft.com/fwlink/?LinkID=863255 for downloading .NET Framework 4.7.2 is not working. This URL is being used in:
   - Line 11
   - Line 28
   - Line 30

I changed all .NET download URLs equal to the URL of the line 26.

Other URL:
 - https://www.microsoft.com/net/download?utm_source=ms-docs&utm_medium=referral (used on line 26)
 - https://dotnet.microsoft.com/download/dotnet-framework-runtime

rpetrusha reviewed (changes requested):
Thank you for noticing and fixing the broken links, @rramoscabral. A better choice, though, is a third link: <https://dotnet.microsoft.com/download/dotnet-framework-runtime/net472>. I suspect the behavior of the "thank you" page link is confusing, and the general .NET Framework download link works for .NET Framework 4.7.2 only as long as it's the most recent version. Could you replace the existing links with <https://dotnet.microsoft.com/download/dotnet-framework-runtime/net472> in all places that include a link to the .NET Framework 4.7.2 download?

rramoscabral committed:
Change .NET Framework 4.7.2 download URL

Change URL  https://www.microsoft.com/net/download/thank-you/net472?utm_source=ms-docs&utm_medium=referral to https://dotnet.microsoft.com/download/dotnet-framework-runtime/net472

rramoscabral commented:
Hi @rpetrusha change the url to https://dotnet.microsoft.com/download/dotnet-framework-runtime/net472

rpetrusha reviewed (approved):
Thanks for making the additional changes, @rramoscabral. I'll merge your PR now. The changes should be live on docs.microsoft.com later today.
